README.md

```diff
@@ -112,17 +109,17 @@ Output:
 
 ```json
 {
-    "hits": [
-        {
-            "id": 1,
-            "title": "Carol",
-            "genre": ["Romance", "Drama"]
-        }
-    ],
-    "offset": 0,
-    "limit": 20,
-    "processingTimeMs": 1,
-    "query": "caorl"
+  "hits": [
+    {
+      "id": 1,
+      "title": "Carol",
+      "genre": ["Romance", "Drama"]
+    }
+  ],
+  "offset": 0,
+  "limit": 20,
+  "processingTimeMs": 1,
+  "query": "caorl"
```

Comment:
My editor seems to have auto-formatted some of these places in the readme when I saved. If you want me to disable my auto-formatting and resave I can do that.

Reply:
that's ok! let's keep it